packages/charts/src/types.ts

```diff
@@ -116,6 +116,7 @@ export type Theme = {
   gridX?: Grid;
   gridY?: Grid;
   labels?: Labels;
+  total?: Labels;
```

Comment:
Please move it - to separate property -> `pie.total` - to keep convention that dedicated theming properties are placed in keys that reflects connection to specific chart.